src/styles/blocks/_review.scss

```diff
@@ -0,0 +1,40 @@
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  align-content: flex-end;
+
+  width: $card-width-without-border * 0.83;
+  margin-bottom: $card-width-without-border * 0.12;
+}
+
+.review__stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.review__star {
+  height: $card-width-without-border * 0.08;
+  width: $card-width-without-border * 0.08;
```

Comment:
it's not intuitive, I need to find that variable and multiply it by 0.08😵‍💫

Reply:
agree